### ISSUE TITLE
Category import removes existing category associations for products

### ISSUE THREAD

**Issue by jameswithers** [dweeves/magmi-git#367]
Not sure if intentional, but category import removes category associations already on products

**Comment by seb4C** [comment]
Yes , it's intentional. Unless explicitely supported by plugins, the magmi behaviour is "take what is in the data source and SET magento item according to it". Some plugins allow "add or differential" syntax but most of magmi processing is done in "replace mode"
